tbx/project_styleguide/templates/patterns/molecules/listing/listing--image.html

```diff
@@ -0,0 +1,42 @@
+{# Designed to be included from another template and values passed in #}
+{% load wagtailimages_tags %}
+
+{% image listing_image fill-370x370 format-webp as desktop_image %}
```

Comment:
It occurs to me that it might be a good idea to give these target vars more specific names as when we start adding more responsive images we'll need to have yaml names that don't clash in the styleguide.

Reply:
I thought the same when doing the standard page! I've given the image block names quite specific names for this reason.

Reply:
Updated in b28870c17ea0ddf220ec32ffd176368ea46eb225